webserver/cWebem.cpp

```diff
@@ -1991,13 +1993,31 @@ void cWebemRequestHandler::handle_request(const request& req, reply& rep)
 	}
 
 	modify_info mInfo;
-	if (!myWebem->CheckForPageOverride(session, requestCopy, rep))
+	if (myWebem->CheckForPageOverride(session, requestCopy, rep))
+	{
+		if (session.reply_status != reply::ok) // forbidden
+		{
+			rep = reply::stock_reply(static_cast<reply::status_type>(session.reply_status));
+			return;
+		}
+
+		if (!rep.bIsGZIP) // this is probably always true?
```

Comment:
Seems like it, please remove the comment

Reply:
and leave the test just in case?

Reply:
From what i tested with breakpoints, at this point it is not compressed
As the boolean is set after compression, i don't expect it to be compressed again
Only some additional headers will be added

Reply:
Didn't verify it. Was just wondering if there could be some action or override that returned already compressed data. I'll leave the test in place.